src/user_templates_api/templates/jupyter_lab/templates/hra_api_tutorial/template.txt

```diff
@@ -0,0 +1,1283 @@
+[
+  {
+    "attachments": {},
+    "cell_type": "markdown",
+    "id": "40dd5488",
+    "metadata": {},
+    "source": [
+      "## HRA-API Documentation"
+    ]
+  },
+  {
+    "cell_type": "code",
+    "execution_count": null,
+    "id": "47b355bd",
+    "metadata": {},
+    "outputs": [],
+    "source": [
+      "# linked datasets\n",
+      "uuids = {{ uuids | safe }}"
+    ]
+  },
+  {
+    "attachments": {},
+    "cell_type": "markdown",
+    "id": "e2d89b95",
+    "metadata": {},
+    "source": [
+      "# INDEX\n",
+      "1. [Installation](#install) \n",
+      "2. [Imports](#imports)\n",
+      "3. [Configuration](#config)\n",
+      "4. [Methods](#methods)\n",
+      "    1. [Database Status](#db_status) \n",
+      "    2. [Run a Sparql query](#sparql_get) \n",
+      "    3. [Run a Sparql query (POST)](#sparql_post) \n",
+      "    4. [Get aggregate results / statistics](#agg_results) \n",
+      "    5. [Given a SpatialEntity already placed relative to a reference SpatialEntity retrieve a new direct SpatialPlacement to the given SpatialEntity IRI](#spatial_entity)\n",
+      "    6. [Get all hubmap rui locations](#rui_loc)\n",
+      "    7. [Get number of ontology term occurrences for a search](#ontology_term) \n",
+      "    8. [Get number of cell type term occurrences for a search](#cell_term_occ) \n",
+      "    9.  [Get ontology term tree nodes](#ontology_nodes) \n",
+      "    10. [Get cell type tree nodes](#cell_nodes)\n",
+      "    11. [Get tissue provider names (for filtering)](#providers) \n",
+      "    12. [Get all reference organs](#ref_organs) \n",
+      "    13. [Get all nodes to form the 3D scene for an organ](#3d_scene) \n",
+      "    14. [Get all nodes to form the 3D scene of reference body, organs, and tissues](#nodes_3d) \n",
+      "    15. [Get technology names (for filtering)](#tech) \n",
+      "    16. [Get Tissue Block Results](#tissue_block) \n",
+      "    17. [Get all GTEx rui locations (if enabled)](#gtex_rui)\n",
+      "    18. [Token](#token_usage) \n",
+      "    19. [Generate a 3D corridor](#corridors)\n",
+      "    20. [Mesh-Based Collision Detection](#collision)\n",
+      "    21. [3D Cell Generation](#3d_cell)\n"
+    ]
+  },
+  {
+    "attachments": {},
+    "cell_type": "markdown",
+    "id": "43773186",
+    "metadata": {},
+    "source": [
+      "<a id='install'></a>\n",
+      "### Installation\n",
+      "\n",
+      "#### hra-api-client can be installed via `pip` from [PyPI](https://pypi.org/project/hra-api-client/#description).\n",
+      "***\n",
+      "> `pip install hra-api-client`\n",
+      "***\n",
+      "\n",
+      "You may also specify the version number:\n",
+      "***\n",
+      "> `pip install hra-api-client==0.10.0`\n",
+      "***"
+    ]
```

Comment:
It's great to show how to install it, but please also include a cell with all the installs. This way, if someone runs the entire notebook, it doesn't halt with an error.

Reply:
Here it would be 
`!pip install hra-api-client==0.10.0 pyvista[jupyter] itkwidgets`